rmldatatfbbox/sample_configs/bbox_all_fields.yaml

```diff
@@ -0,0 +1,19 @@
+# This sample config contains all fields supported by ravenML core and the bbox plugin.
+# Plugin specific configuration is located in the plugin field.
+
+imageset: # Can provide a list of imagesets if needed
+  - jigsaw_tester
```

Comment:
If one wants to take images from multiple imagesets it is supported by just adding more lines with the prefixes